hpedockerplugin/file_manager.py

```diff
@@ -652,9 +654,28 @@ def mount_share(self, share_name, share, mount_id):
                 int(fMode)
                 chmod(fMode, mount_dir)
             except ValueError:
-                self._hpeplugin_driver.set_ACL(fMode, mount_dir, fUName,
-                                               fGName)
-
+                fUserId = share['id']
+                try:
+                    self._hpeplugin_driver.set_ACL(fMode, fUserId, fUName,
+                                                   fGName)
+                except exception.ShareBackendException as ex:
+                    msg = (_("Exception raised for ACL setting,"
+                             " but proceed. User is adviced to correct"
+                             " the passed fsMode to suit its owner and"
+                             " group requirement. Delete the share and "
+                             " create new with correct fsMode value."
+                             " Please also refer the logs for same. "
+                             "Exception is  %s") % six.text_type(ex))
+                    LOG.info(msg)
+                    LOG.info("Unmounting the share,permissions are not set.")
```

Comment:
this should be an error message `LOG.error`

Reply:
Changed LOG.info to LOG.error  I have kept LOG.info for next line to inform user because of error, we are unmounting and  removing direcory.

Reply:
@wdurairaj Done.